service/service.pro

```diff
@@ -6,18 +6,25 @@ DISTFILES += \
     dbus-org.sailfishos.usermanager.service \
     org.sailfishos.usermanager.conf \
     org.sailfishos.usermanager.service \
-    home-sailfish_guest.mount
+    home-sailfish_guest.mount \
+    org.sailfishos.usermanager-late.conf \
+    org.sailfishos.usermanager-late@.service
```

Comment:
Could we name this differently? Perhaps `session-startup-scripts@.service` or something like that. Having user-managerd involved there is just an implementation detail and not something that defines the service IMO.